ethereum/contracts/Starport.sol

```diff
@@ -13,9 +13,9 @@ contract Starport {
     ICash immutable public cash;
 
     address immutable public admin;
-    address constant public ETH_ADDRESS = address(0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE);
-    bytes4 constant MAGIC_HEADER = "ETH:";
-    string constant ETH_CHAIN = "ETH";
+    address constant public NATIVE_TOKEN_ADDRESS = address(0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE);
```

Comment:
This maybe should be immutable and different for different chains, e.g. `0xAaAAaaaaaAAAaaaAAaaaaaaAaAaaaAa` for mAtic?

Reply:
i think i'm ok with the native token address being the same across all evm chains - don't really see any reason why they should be different